subsys/bluetooth/host/keys.c

```diff
@@ -51,7 +59,24 @@ struct bt_keys *bt_keys_get_addr(u8_t id, const bt_addr_le_t *addr)
 			first_free_slot = i;
 		}
 	}
-
+#if defined(CONFIG_BT_KEYS_OVERWRITE_OLDEST)
+	if (first_free_slot == ARRAY_SIZE(key_pool)) {
+		size_t oldest_slot = 0;
+
+		for (i = 1; i < ARRAY_SIZE(key_pool); i++) {
+			if (key_pool[oldest_slot].aging_counter
+			    > key_pool[i].aging_counter) {
```

Comment:
Move the `>` to the first line, so that the line split happens after it (same rule applies for other operators as well when you need to split a line). Alternatively (possibly better), add a helper variables here for the `i` and `oldest_slot` entries (e.g. `struct bt_keys *oldest, *current`) to shorten the line and avoid a split altogether - the compiler will probably anyway generate the same code.